sdk/core/Microsoft.Azure.Core.Spatial/src/Microsoft.Azure.Core.Spatial.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyTitle>System.Text.Json support for the Microsoft.Spatial library</AssemblyTitle>
+    <Description>This library contains converters for System.Text.Json for use with Microsoft.Spatial when using the Azure SDK for .NET.</Description>
+    <RootNamespace>Azure.Core</RootNamespace>
+    <Version>1.0.0-preview.1</Version>
+    <PackageTags>Microsoft Azure SDK Spatial System Text Json</PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <EnableClientSdkAnalyzers>false</EnableClientSdkAnalyzers>
+  </PropertyGroup>
+
+  <Import Project="..\..\Azure.Core\src\Azure.Core.props" />
```

Comment:
Let's remove this for now.

Reply:
I need to refer to it in Search. This will make any use of it easy to switch to a PackageReference later.